docs/source/software/conda.rst

```diff
@@ -15,14 +15,15 @@ Creating a Conda virtual environment with Anaconda
 Using a locally installed Conda virtual environment is highly recommended so that you can install the specific packages that you need.
 You can also have more than one environment with different packages for testing purposes. This procedure uses the Anaconda module already loaded on Discovery.
 
-1. To check what version of Python you have installed, type ``which python``.
-2. To load anaconda, type ``module load anaconda3/2022.01``.
-3. To create your environment, type ``conda create -n <yourenvironmentname> python=3.7 anaconda``, where <yourenvironmentname> is the name you want to give your environment. Tip: to see a list of all of your conda environments, type ``conda info -e``. To save space, also use the ``--prefix=/work/<mygroup>/<mydirectory>`` flag to build in your work directory.
-4. Follow the prompts to complete the Conda install.
-5. To activate your Conda environment, type ``source activate <yourenvironmentname>``. Note that ``conda activate`` will not work on Discovery with this version.
-6. To install a specific package, type ``conda install -n <yourenvironmentname> [package]``.
-7. To deactivate the current, active Conda environment, type ``conda deactivate``.
-8. To delete a Conda environment and all of its related packages, type ``conda remove -n <yourenvironmentname> --all``.
+1. If you are on a login node, move to a compute node by typing ``srun --partition=short --nodes=1 --cpus-per-task=1 --pty /bin/bash``. Allocate resources as per your requirements.
```

Comment:
Before starting with "allocate resources...", you can say something like:
"In the above example, we request for 1 node and 1 cpu, but you can request for additional resources as per your requirements.

Reply:
@ShobanaSekar  I have made the changes you mentioned and have tested it out.